l10n_br_purchase/hooks.py

```diff
@@ -3,6 +3,8 @@
 
 from odoo import SUPERUSER_ID, api, tools
 
+from odoo.addons.l10n_br_fiscal.tools import set_journal_in_fiscal_operation
+
 
 def post_init_hook(cr, registry):
     cr.execute("select demo from ir_module_module where name='l10n_br_purchase';")
```

Comment:
please change this line to env.ref('base.module_l10n_br_purchase').demo, would be cool if you could update in the other places as well. Seems cleaner, ripped from Odoo: https://github.com/odoo/odoo/blob/25feb5b11c2df401580b65e0108145863fcf8987/addons%2Fl10n_ar_withholding%2F__init__.py#L34C16-L34C63

Reply:
Done, change made in l10n_br_sale, l10n_br_purchase, l10n_br_stock and l10n_br_stock_account